app/assets/javascripts/controllers/automation_manager/automation_manager_form_controller.js

```diff
@@ -4,21 +4,19 @@ ManageIQ.angular.app.controller('automationManagerFormController', ['$http', '$s
   vm.automationManagerModel = angular.copy(configurationManagerService.managerModel);
 
   vm.postValidationModel = {};
-
-  vm.formId = automationManagerFormId;
```

Comment:
There's a few places which still look at `formId`, `CredentialsController` being one of them.. maybe shouldn't remove it?